### ISSUE TITLE
GPU supported databases

### ISSUE THREAD

**Issue by rukibuki** [soedinglab/MMseqs2#908]
I was just wondering if any databases have been converted to support GPU already? I have converted uniref90 myself following the small section on that here on your github, but if you had already made all/many of the databases available in a GPU version I would much prefer to dl those.

**Comment by milot-mirdita** [comment]
Currently, we haven't started any plans on prebuilt GPU databases (yet). I agree that this would be useful.

Currently, you have to call `makepaddedseqdb` on the target database after a `createdb` or `databases` call.
See https://github.com/soedinglab/MMseqs2/wiki#gpu-accelerated-search

**Comment by rukibuki** [comment]
yes that is exactly what I have done so far on uniref90, and it took some time to process that database.  I am just unsure which database (or databases) to convert like that, if I want to use it/them for MSA generation for an AF3 pipeline afterwards. any suggestions?

**Comment by milot-mirdita** [comment]
We plan to make MMseqs2-GPU databases available for ColabFold, which we also plan to optimize to AF3 (or to some of the free alternatives) at some point soon.

**Comment by brianloyal** [comment]
FWIW, let me know if you're interested in hosting these optimized databases on AWS. We have a [program](https://aws.amazon.com/opendata/open-data-sponsorship-program/) to sponsor high-impact scientific data sets (e.g. https://registry.opendata.aws/openfold/) and this would def count

**Comment by martin-steinegger** [comment]
@brianloyal it would be fantastic to host the database on AWS. I recall applying to AWS in the past, but unfortunately, we were rejected. Do you have any recommendations to improve our chances this time?
